src/rules/utils.ts

```diff
@@ -1,10 +1,10 @@
-import { parse as parsePath } from 'path';
 import {
   AST_NODE_TYPES,
   ESLintUtils,
   TSESLint,
   TSESTree,
 } from '@typescript-eslint/experimental-utils';
+import { parse as parsePath } from 'path';
```

Comment:
they were sorted as node core at the top

Reply:
Ah fair enough - I was using WebStorms optimize imports function, which sadly doesn't support that sort of logic.

Having it as an eslint rule should take care of that :)